utils/dom.js

```diff
@@ -307,7 +284,7 @@ export function placeCaretAtVerticalEdge( container, isReverse, rect, mayUseScro
 	// equivalent to a point at half the height of a line of text.
 	const buffer = rect.height / 2;
 	const editableRect = container.getBoundingClientRect();
-	const x = rect.left + ( rect.width / 2 );
+	const x = rect.left;
```

Comment:
Just curious: what prompted this change?

Reply:
When pressing up from an uncollapsed selection, we want the caret to move directly above where the start of the selection occurs. Previously, it would place it above the half-way point of the selection.

![image](https://user-images.githubusercontent.com/1779930/39643417-b52a0792-4fa2-11e8-95f0-d081f0c41993.png)